modules/private/internal.go

```diff
@@ -0,0 +1,51 @@
+package private
```

Comment:
Why is it called `private` when then endpoint is `/internal` ? Pick one and stay with it 😉